cloud/openstack/os_user.py

```diff
@@ -219,6 +215,10 @@ def main():
             domain_id = _get_domain_id(opcloud, domain)
 
         if state == 'present':
+            if update_password in ('always', 'on_create'):
+                if not password:
+                    msg = ("update_password is %s but a password value is "
```

Comment:
Lol, good catch, I was a bit too hurried this morning. I intended to use it in a `fail_json` call, which I have just done with a newer change set.